nephio-ansible-install/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM us-central1-docker.pkg.dev/cloud-workstations-images/predefined/code-oss@sha256:eae70293915c10aae4909c407eaa0a59a813bbab9a21667fdbf0e72502b28c3e
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y python3=3.9.2-3 python3-docker=4.1.0-1.2 && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN wget -q https://open-vsx.org/api/redhat/vscode-yaml/1.10.1/file/redhat.vscode-yaml-1.10.1.vsix && \
+unzip redhat.vscode-yaml-1.10.1.vsix "extension/*" && \
+mv extension /opt/code-oss/extensions/redhat.vscode-yaml
+
+RUN wget -q https://open-vsx.org/api/ms-kubernetes-tools/vscode-kubernetes-tools/1.3.11/file/ms-kubernetes-tools.vscode-kubernetes-tools-1.3.11.vsix && \
+unzip ms-kubernetes-tools.vscode-kubernetes-tools-1.3.11.vsix "extension/*" && \
+mv extension /opt/code-oss/extensions/ms.kubernetes-tools
+
+RUN mkdir -p /nephio-installation/inventory
+COPY ansible.cfg /nephio-installation/
```

Comment:
curious as to why the nind related files are in the nephio-ansible-install folder instead of being in its own folder (i.e sibling to nephio-ansible-install directory).

Reply:
Quite often Dockerfiles are sitting in the same folder as the application. Reckoned it made sense to surface it there also for nind. Could move the assets to their own folder in case you think it makes sense?

Reply:
I have seen go projects using `/build/package` directory or similar for Dockerfiles

Reply:
Good point and see your point @vjayaramrh
IMHO in our case I think the Dockerfile is in a suitable location in the nephio-ansible-install folder.
Like this the folder could serve as a multiple purpose one stop shop depending on how an implementer would like to install nephio.

1. Installing nephio by by running the ansible playbooks on a suitable VM (as exercised in the workshop)
2. In a complete end-to-end integrated containerized fashion with nind (by simply building and executing the container specified in the Dockerfile)
3. Somewhat/somewhere in between installing nephio by runnning the ansible playbooks in a suitable (base)container image (this is still work in progress but @henderiw and i discuss on a follow up PR for this)

This would give an implementer maximum flexibility on how they choose to explore nephio.

Hence, long story short I think the Dockerfile is well positioned in this folder as it fits in well with the storyline above and also makes it easily discoverable.
Apologies for the wall of words but thought it might help outline the reasoning behind the choice.